python/source/create.rst

```diff
@@ -67,6 +67,58 @@ from a variety of inputs, including plain python objects
     :func:`pyarrow.array` for conversion to Arrow arrays,
     and will benefit from zero copy behaviour when possible.
 
+Creating RecordBatches
```

Comment:
Nit: Given the titles are prose I think it might be more correct to separate the words.
```suggestion
Creating Record Batches
```